src/exporter/exporter-sparql.js

```diff
@@ -30,6 +34,17 @@ export class ExporterSparql extends Exporter {
             });
     }
 
+    async superAdminExists(client) {
+        let results;
+        await client.query("SELECT ?superAdmin WHERE {?superAdmin <" + ONTOLOGY_IRI + "isSuperAdmin> true}")
+            .then((r) => {
+                results = r.results.bindings;
+            }).catch((e) => {
+                console.log(e);
+            });
+        return results && results.length > 0;
```

Comment:
Zbytočne miešate promises a async/await a ak nastane výnimka, používate neinicializovanú premennú `results`. Keď používate await, chyba sa odchytáva ako výnimka:
```javascript
try {
    const results = await client.query(…).results.bindings;
    return results && results.length > 0;
} catch (e) {
    console.log(e);
    return false;
}
```

Reply:
await client.query(…).results.bindings;  treba pridat aj check na results a results.bindings inak to pada